pandas/plotting/_core.py

```diff
@@ -1,6 +1,7 @@
 from __future__ import annotations
 
 import importlib
+from importlib.metadata import entry_points
```

Comment:
this add 30ish ms to import time

Reply:
@Thextan let's add `importlib.metadata` to the BLOCKLIST as well

Reply:
Reverted code back.